assets/css/leaderboard.css

```diff
@@ -146,3 +147,55 @@ th:not(.detail-header)  {
     background-color: #000000; /* Darker shade of blue on hover */
     transform: scale(1.05); /* Slightly increase size on hover */
 }
+
+thead {
+  position: sticky;
+  top: 0;
+  background-color: #f1f8ff; /* Background for better visibility */
+  background-clip: border-box;
+  z-index: 100; /* Ensure the header stays above other content */
+  margin: 0;
+}
+
+#leaderboard thead th{
+    z-index: 100; /* Ensure the header stays above other content */
+    /* border-bottom: 1px solid black;  */
+    background-color: #f1f8ff; /* Background for better visibility */
+}
+
+/* Sticky left columns */
```

Comment:
Can we disable the frozen column when it's rendered in mobile view (check for view width)? Because mobile devices are just so narrow, and after the frozen column, users can barely get any useful information from them (see screenshot). 
Frozen header is enabled in both laptop and mobile view (not affected). 
<img width="346" alt="image" src="https://github.com/user-attachments/assets/7cd16475-890d-4460-b5ba-1167b4a5a5f9">

Reply:
Yes for sure. I'll make the adjustment.